docs/guides/templates/profile.adoc

```diff
@@ -0,0 +1,11 @@
+<#macro ifProduct>
+ifdef::project_product[]
```

Comment:
In other places we use `ifeval::[{project_product}==true]`, do you think it would make sense to make it consistent?

Reply:
Done